MQTicketService/Dockerfile

```diff
@@ -21,21 +27,46 @@ COPY ./TicketGenerator /data/TicketGenerator
 RUN cd /data/TicketGenerator && \
     mvn -f /data/TicketGenerator/pom.xml clean package;
 #
+# stage 2
+#
 #
 # Switch to the MQ image
-# Tested with
+# Tested with 
 #     9.2.5.0-r2
 #     9.3.0.0-r1
 #
-FROM icr.io/ibm-messaging/mq:latest
+FROM $imageName
+#
+# Specify platform architecture - either arm64 or amd64(default)
+ARG platformArch=amd64
+#
+# Set platform architecture as an environment variable
+ENV platformArch=${platformArch}
+#
 USER 0
-
-RUN curl -Lo OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz && \
+#
+#
+RUN if [ "$platformArch" == "amd64" ]; then \
+    curl -Lo OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz && \
```

Comment:
paramatize `jre_x64_linux_hotspot_11.0.13_8` and `11.0.13_8` so they aren't repeated, and only set once.

Reply:
rather like the arm64 version